content/posts/10-year-anniversary.md

```diff
@@ -0,0 +1,65 @@
+title: 10 years of attempting to make open source sustainable
+date: 2024-08-15
+description: Reflecting on 10 years of trying to make open source sustainable
+category: Meta
+tags: meta, open-source, sustainability, anniversary
+authors: Eric Holscher
+status: published
+image: /images/10-year.jpg
+image_credit: Photo by <a href="https://unsplash.com/@adigold1?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Adi Goldstein</a> on <a href="https://unsplash.com/photos/selective-focus-photography-of-assorted-color-balloons-Hli3R6LKibo?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>
+
+Read the Docs was started at a [48-hour open source sprint in 2010](https://www.ericholscher.com/blog/2010/aug/16/announcing-read-docs/), and grew to become a large part of the Python documentation ecosystem within a few years.
+Based on that success and the stress that came with it, we [created a company](https://www.ericholscher.com/blog/2014/oct/24/announcing-read-the-docs-for-business/) to focus on sustainability in 2014.
+We've been working on Read the Docs as a project for 14 years, and a company for 10 years.
+This post will reflect on the last 10 years of trying to make open source sustainable.
+
+## Our sustainability model
+
+We tried a number of different ways to make a sustainable open source project, really a service, over the years.
+The model that we've found that has worked for us currently is a version of the classic open source model used by GitHub, Travis CI, and others:
```

Comment:
I removed the examples, since I feel like nobody else is really doing quite the same thing -- I do think Travis did it the most with the 2 sites running on OSS, but not sure of other examples (not sure that's a good thing 😆)

Reply:
Yeah, we definitely were following Travis' lead for sure. It only feels sad comparing because of their PE buyout and all of the fallout there.